src/components/KYCWall/BaseKYCWall.tsx

```diff
@@ -246,7 +247,7 @@ function KYCWall({
         <>
             <AddPaymentMethodMenu
                 isVisible={shouldShowAddPaymentMenu}
-                iouReport={iouReport}
+                iouReport={isEmptyObject(iouReport) ? undefined : iouReport}
```

Comment:
Maybe we can update `iouReport` type to include `EmptyObject` in the `AddPaymentMethodMenu` instead?